### PR TITLE
Notification component

### PR DESCRIPTION
**Pull request by dyang5200**
Resolves #

### CODE REVIEW

File: frontend/src/styles/index.scss [hunk 1]
```diff
@@ -23,6 +23,14 @@ body {
...
       margin: 1%;
     }
+
+    .name {
+      color: #4FAF4E
+    }
+
+    .description {
+      color: #484848
+    }
```
Comment (rgychiu) [inline frontend/src/styles/index.scss]: Another CSS LEGEND in the making. Some comments about the CSS atm though is that generally the index.scss/index.css file is used for more general classes/styling. Classes that are very broad and can be reused across multiple components are kept at this level as a result (some examples are like .text-center, which can apply in a lot of different cases in many different css files and classes).

For more specific styling or CSS targets that apply specifically to one component, another SCSS file should be made. So in this case you would probably want another SCSS file called notification.scss.

Also SCSS has a very neat feature of being able to define variables and use/import them throughout the rest of the SCSS files, so I would consider looking into making a variables.scss file that defines all of your colors and then importing them in other SCSS files. If you need some examples with that, I did a little bit of that in the dashboard-styling branch, so feel free to check that out :))

File: frontend/src/components/Notification.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+class Notification extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      name: this.props.name,
+      time: this.props.time,
+      description: this.props.description
+    }
```
Comment (arpanlaha) [inline frontend/src/components/Notification.js]: do we need to set props to state or can we just use this.props._____

File: frontend/src/styles/notification.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+// Different from notif-description in case later
+// we need differing styles for the time vs. description
+.time {
+  color: $gray_text_color;
+}
+
+.notif-description {
+  color: $gray_text_color;
+}
```
Comment (rgychiu) [inline frontend/src/styles/notification.scss]: NICE

File: frontend/src/components/Notification.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    super(props)
+  }
+
+  render() {
+    const { isPM } = this.props
+    return (
+      <ListGroupItem>
+        <p className="person-name">
+          {' '}
+          <b> {this.props.name} </b>{' '}
```
Comment (rgychiu) [inline frontend/src/components/Notification.js]: I can't think of a better way to do this right now, but I feel like there might be a cleaner way to include spaces here. We'll probably need to look at it later though.

**Pull request merged**